components/eam/src/physics/cam/hetfrz_classnuc_cam.F90

```diff
@@ -1617,7 +1617,7 @@ subroutine get_aer_num(ii, kk, ncnst, aer, aer_cb, rhoair,&
       end if
 
       if (awcam(3) > 0._r8) then
-#if (defined MODAL_AERO_4MODE_MOM && defined RAIN_EVAP_TO_COARSE_AERO )
+#if ( (defined MODAL_AERO_4MODE_MOM || defined MODAL_AERO_5MODE) && defined RAIN_EVAP_TO_COARSE_AERO )
```

Comment:
@mingxuanwupnnl do you want to change line 1625 #elif (defined MODAL_AERO_4MODE_MOM) to #elif (defined MODAL_AERO_4MODE_MOM || defined MODAL_AERO_5MODE) ? Otherwise, MODAL_AERO_5MODE without RAIN_EVAP_TO_COARSE_AERO would be #else

Reply:
@xuezhengllnl Thank you. I added MODAL_AERO_5MODE for places where there is only MODAL_AERO_4MODE_MOM. Note that both MODAL_AERO_5MODE and RAIN_EVAP_TO_COARSE_AERO are always on by default for v3.